docs/setup/jetson.md

```diff
@@ -154,40 +299,30 @@ The recommended way to run ROS2 on Jetson is to use pre-built Docker container i
         /wheel_ticks
         /wheel_vels
 
-    Check if it read a message on a topic
+1. Check if it reads a message on a topic
 
-        ros2 topic echo odom
+        ros2 topic echo battery_state
 
     You should see continuous repetition of output like following.
```

Comment:
"like the following"

Reply:
[d75637a](https://github.com/iRobotEducation/create3_docs/pull/385/commits/d75637a19e07004cf2e2c4e3c6f5ce11f0ff70aa)